src/main/java/io/debezium/connector/vitess/connection/VStreamOutputMessageDecoder.java

```diff
@@ -78,7 +78,7 @@ public void processMessage(Binlogdata.VEvent vEvent, ReplicationMessageProcessor
             case VERSION:
                 break;
             default:
-                LOGGER.warn("VEventType {} skipped, not processing.", vEventType);
+                LOGGER.info("VEventType {} skipped, not processing.", vEventType);
```

Comment:
Minor thing - why change this to info? We might lose this message depending on log level.

Reply:
Warning level is too high. We currently see a lot of SAVEPOINT in logs (500/s), so I had to change admin logger to change log level to "error" (above "warning") to disable this log.

By changing it to "info", we only need to override it to "warn", giving our warning log lines in this class an opportunity to show up in the log